### PR TITLE
IESG review update

### PR DESCRIPTION
**Pull request by vb-balazs**
IESG review comments resolved in this version

### CODE REVIEW

**Comment by vb-balazs** [comment]
MPLSoUDP

File: mpls-over-udp-ip/draft-ietf-detnet-mpls-over-udp-ip.xml [hunk 1]
```diff
@@ -152,6 +152,7 @@
  <t>
   These requirements are satisfied by the DetNet over MPLS Encapsulation
   described in <xref target="I-D.ietf-detnet-mpls"/> and they are partly satisfied
+  (requirement 1 and 4)
```
Comment (louberger) [inline mpls-over-udp-ip/draft-ietf-detnet-mpls-over-udp-ip.xml]: perhaps:
, i.e., IP flows can be identified however no DetNet sequence number is carried,

File: mpls-over-udp-ip/draft-ietf-detnet-mpls-over-udp-ip.xml [hunk 2]
```diff
@@ -152,6 +152,7 @@
  <t>
```
Comment (vb-balazs) [inline mpls-over-udp-ip/draft-ietf-detnet-mpls-over-udp-ip.xml:152]: A method for queuing and forwarding indication.

File: mpls-over-udp-ip/draft-ietf-detnet-mpls-over-udp-ip.xml [hunk 3]
```diff
@@ -250,8 +251,7 @@
   <t>
     S-Labels, d-CW and zero or more F-Labels are used as defined in <xref
     target="I-D.ietf-detnet-mpls"/> and are not modified by this document.
-    In case of aggregates the A-Label is treated as an S-Label and it
-    too is not modified.
+    In case of aggregates the A-Label (if present) is not modified as well.
```
Comment (louberger) [inline mpls-over-udp-ip/draft-ietf-detnet-mpls-over-udp-ip.xml]: how about drop sentence and change to " "S-Labels, A-Labels (when present), d-CW..."

File: mpls-over-udp-ip/draft-ietf-detnet-mpls-over-udp-ip.xml [hunk 4]
```diff
@@ -263,17 +263,18 @@
...
+    target="RFC7510"/>, and the 
+	UDP Source Port value MUST be set to uniquely identify the DetNet flow.
+	The packet MUST then be handled
     as a DetNet IP packet, per <xref target="I-D.ietf-detnet-ip"/>.
     This includes QoS related traffic treatment.
   </t>
 
   <t>
-    To support receive processing an implementation MUST also support
+    To support processing incoming DetNet MPLS over UDP/IP encapsulation, an
```
Comment (louberger) [inline mpls-over-udp-ip/draft-ietf-detnet-mpls-over-udp-ip.xml]: How about: To support the receive processing defined in this document

File: mpls-over-udp-ip/draft-ietf-detnet-mpls-over-udp-ip.xml [hunk 5]
```diff
@@ -289,23 +290,24 @@
             The following summarizes the set of information that is needed to
             configure DetNet MPLS over UDP/IP:
             <list style="symbols">
-                          <t>Label information (S-label or F-label) to
-                          be mapped to UDP/IP flow. Note that a single
-                          S-Label can map to multiple sets of UPD/IP
+                          <t>Label information (A-label, S-label or F-label) to
```
Comment (louberger) [inline mpls-over-udp-ip/draft-ietf-detnet-mpls-over-udp-ip.xml]: s/label/labels/g

Comment (louberger) [reply mpls-over-udp-ip/draft-ietf-detnet-mpls-over-udp-ip.xml]: s/or/and/

**Review by louberger — DISMISSED** [review]
can be merged once comments integrater, per Jan 27th meeting

**Comment by vb-balazs** [comment]
MPLSoUDP: all 01-27 changes done.

**Review by jfarkas — APPROVED** [review]

**Pull request merged**